### PR TITLE
INF-836: Fix flaky timeout issue

### PR DESCRIPTION
**Pull request by wivorn**
# [INF-836](https://linear.app/superhuman/issue/INF-836/flaky-zen-test-navigation-timeout-exceeds-30000)

- Fix timeout issue on Zen tests
- Constraint Zen UI to only move horizontally

Reference: [https://ourcodeworld.com/articles/read/1106/how-to-solve-puppeteer-timeouterror-navigation-timeout-of-30000-ms-exceeded](https://ourcodeworld.com/articles/read/1106/how-to-solve-puppeteer-timeouterror-navigation-timeout-of-30000-ms-exceeded)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# 2022-5-5 0.3.21
```
Comment (Timothee) [inline CHANGELOG.md:1]: Just flying by: we're in 2023 😋

File: lib/chrome_wrapper.ts [hunk 2]
```diff
@@ -522,6 +522,8 @@ export default class ChromeWrapper {
 
     const browser = await this.browser
     const page = await browser.newPage()
+    await page.setDefaultNavigationTimeout(0)
```
Comment (zindlerb) [inline lib/chrome_wrapper.ts:525]: Hmm I wonder if we should just make the timeout really long like 10min. Could removing the timeout lead to tests running forever in ci?

**Review by zindlerb — APPROVED** [review]
lgtm

**Review by zindlerb — APPROVED** [review]
lgtm

**Pull request closed without merging**